src/3rdParty/CMakeLists.txt

```diff
@@ -13,3 +13,12 @@ if (BUILD_ASSEMBLY AND NOT FREECAD_USE_EXTERNAL_ONDSELSOLVER)
     endif()
     add_subdirectory(OndselSolver)
 endif()
+
+if (BUILD_TRACY_FRAME_PROFILER)
+    if( NOT EXISTS "${CMAKE_SOURCE_DIR}/src/3rdParty/tracy/CMakeLists.txt" )
+        message( SEND_ERROR "The Tracy git directory is not available. Please clone it manually." )
```

Comment:
Is using `SEND_ERROR` here considered a best practice in the CMake world? I don't really like errors that cause generation to fail, but then you have to scroll back up through the whole process to find what went wrong. IMO if it's fatal, make it a `FATAL_ERROR`. But if `SEND_ERROR` is considered "the right way" then I guess I will adapt :).